deep_research/requirements.txt

```diff
@@ -1,11 +1,11 @@
 zenml>=0.82.0
-litellm>=1.70.0,<2.0.0
+litellm>=1.73.0
 tavily-python>=0.2.8
 exa-py>=1.0.0
 PyYAML>=6.0
 click>=8.0.0
 pydantic>=2.0.0
 typing_extensions>=4.0.0
 requests
-langfuse>=2.0.0
+langfuse<3.0.0
```

Comment:
This is an unneccessary pinnning - the only reason this breaks for langfuse > 3.0 is that we are importing (quite needlessly IMO) some type hints. Rather remove the type hints or move the type hint classes to If TYPE_CHECKING and it will be solved

Reply:
Also this isn't the only problem. If you try running with >3.0 langfuse and latest litellm, it doesn't work. the litellm autologger is using <3.0 syntax for what it does, I think. I'll return to this in the morning but not sure this can be made to work with >3.0 langfuse installed.